src/Kros.AspNetCore/Properties/Resources.resx

```diff
@@ -120,6 +120,9 @@
   <data name="AuthorizationServiceForbiddenRequest" xml:space="preserve">
     <value>Authorization service forbidden this request.</value>
   </data>
+  <data name="BadRequest" xml:space="preserve">
+    <value>User request is invalid</value>
```

Comment:
Nit: can you add a dot to the end of a sentence? Thanks.
```suggestion
    <value>User request is invalid.</value>
```

Reply:
Here your are.